File: modul-1/PRAK110-2510817220019-TysaLuthfia.c
#include <stdio.h> 
#include <math.h>

int main()
{
    int al = 5; 
    int ti = 12; 
    int sa = ti;
    int sb = sqrt(al * al + ti * ti); 
    int sc = al;
    int kel= (sa + sb + sc); 
    int lu = (al * ti) / 2;
    printf("Diketahui :\n"); 
    printf("Alas = %d cm\n", al);
    printf("tinggi = %d cm\n", ti); 
    printf("\n");
    printf("Jawab :\n"); 
    printf("Sisi A = %d cm\n", sa); 
    printf("Sisi B = %d cm\n", sb); 
    printf("Sisi C = %d cm\n", ti);
    printf("Keliling = %d cm\n", kel); 
    printf("Luas = %d cm\n", lu); 
    return 0;
}

File: modul-1/PRAK110-2510817220019-TysaLuthfia.py
import math
al= 5
ti= 12
sa= ti
sb = math.sqrt(al * al + ti * ti) 
sc = al
kel = (sa + sb + sc)
lu = (al * ti) /2
print("Diketahui :")
print("Alas = {} cm " .format(al)) 
print("Tinggi = {} cm\n" .format(ti)) 
print("Jawab :")
print("Sisi A = {} cm" .format(ti)) 
print("Sisi B = {} cm" .format(round(sb))) 
print("Sisi C = {} cm" .format(al)) 
print("Keliling = {} cm" .format(round(kel)))
print("Luas = {} cm" .format(round(lu)))